nextcloudappstore/core/fixtures/categories.json

```diff
@@ -129,6 +129,16 @@
             "master": "workflow"
         }
     },
+    {
+        "model": "core.categorytranslation",
+        "pk": 15,
+        "fields": {
+            "language_code": "en",
+            "name": "Dashboard",
+            "description": "Apps for Nextcloud Dashboard",
```

Comment:
I think this category is for apps that "include" a dashboard widget but that are potentially not specialized in that area. What about changing this to
`Apps including Nextcloud Dashboard widgets` ?